doc/devguide/codebase/code-style.rst

```diff
@@ -159,7 +460,59 @@ TODO
 Macros
 ~~~~~~
 
-TODO
+Macro names are ALL_CAPS_WITH_UNDERSCORES.
+Enclose parameters in parens on each usage inside the macro.
+
+Align macro values on consecutive lines.
+
+.. code-block:: c
+
+    #define ACTION_ALERT       0x01
+    #define ACTION_DROP        0x02
+    #define ACTION_REJECT      0x04
+    #define ACTION_REJECT_DST  0x08
+    #define ACTION_REJECT_BOTH 0x10
+    #define ACTION_PASS        0x20
+
+Align escape for multi-line macros left-most.
```

Comment:
Some code does right align the backslash for macros, e.g. queue.h, some does not align, e.g. util-spm.h.

The question is what backslash alignment do you guys prefer.

Reply:
Right align. I think it's better for readability, even if it makes editing a bit tedious.

Reply:
Editing is easy. Don't bother with alignment when editing and simply run git clang-format.